com.unity.netcode.gameobjects/Runtime/Serialization/FastBufferReaderExtensions.cs

```diff
@@ -0,0 +1,327 @@
+                    arr.SetValue(item, i);
+                }
+
+                value = arr;
+                return;
+            }
+
+            if (type.IsEnum)
+            {
+                switch (Type.GetTypeCode(type))
+                {
+                    case TypeCode.Boolean:
+                        reader.ReadValueSafe(out byte boolVal);
+                        value = Enum.ToObject(type, boolVal != 0);
+                        return;
+                    case TypeCode.Char:
+                        reader.ReadValueSafe(out char charVal);
+                        value = Enum.ToObject(type, charVal);
+                        return;
+                    case TypeCode.SByte:
+                        reader.ReadValueSafe(out sbyte sbyteVal);
+                        value = Enum.ToObject(type, sbyteVal);
+                        return;
+                    case TypeCode.Byte:
+                        reader.ReadValueSafe(out byte byteVal);
+                        value = Enum.ToObject(type, byteVal);
+                        return;
+                    case TypeCode.Int16:
+                        reader.ReadValueSafe(out short shortVal);
+                        value = Enum.ToObject(type, shortVal);
+                        return;
+                    case TypeCode.UInt16:
+                        reader.ReadValueSafe(out ushort ushortVal);
+                        value = Enum.ToObject(type, ushortVal);
+                        return;
+                    case TypeCode.Int32:
+                        reader.ReadValueSafe(out int intVal);
+                        value = Enum.ToObject(type, intVal);
+                        return;
+                    case TypeCode.UInt32:
+                        reader.ReadValueSafe(out uint uintVal);
+                        value = Enum.ToObject(type, uintVal);
+                        return;
+                    case TypeCode.Int64:
+                        reader.ReadValueSafe(out long longVal);
+                        value = Enum.ToObject(type, longVal);
+                        return;
+                    case TypeCode.UInt64:
+                        reader.ReadValueSafe(out ulong ulongVal);
+                        value = Enum.ToObject(type, ulongVal);
+                        return;
+                }
+            }
+
+            if (type == typeof(GameObject))
+            {
+                reader.ReadValueSafe(out GameObject go);
+                value = go;
+                return;
+            }
+
+            if (type == typeof(NetworkObject))
+            {
+                reader.ReadValueSafe(out NetworkObject no);
+                value = no;
+                return;
+            }
+
+            if (typeof(NetworkBehaviour).IsAssignableFrom(type))
+            {
+                reader.ReadValueSafe(out NetworkBehaviour nb);
+                value = nb;
+                return;
+            }
+            /*if (value is INetworkSerializable)
```

Comment:
👀

Reply:
Removed this whole class. No support for serializing by object anymore - nothing needs it internally so it's gone.